xformers/components/positional_embedding/sine.py

```diff
@@ -4,32 +4,45 @@
 # LICENSE file in the root directory of this source tree.
 
 
+# Silence Mypy errors in this file.
```

Comment:
We are still using Mypy, which means it complains about sine.py since it doesn't understand variadics yet.

So, I've made Mypy ignore the entire file. Long-term, if we move off Mypy, we can remove this.